README.md

```diff
@@ -0,0 +1,165 @@
+```
+# [in python, inside the LSTM_temp2 directory]
+import os
+import sciencebasepy
+from zipfile import ZipFile
+sb = sciencebasepy.SbSession()
+sb.login(username, password) # enter your username and password. this is only necessary before the data are made public
+os.mkdir('datarelease')
+sb.get_item_files(sb.get_item('606db85fd34e670a7d5f61f0'), 'datarelease') # basins and gages
+sb.get_item_files(sb.get_item('6083384fd34efe46ec0a2333'), 'datarelease') # temperature observations
+sb.get_item_files(sb.get_item('6084cab2d34eadd49d31aeab'), 'datarelease') # model drivers and basin attributes
+sb.get_item_files(sb.get_item('6084cb2ed34eadd49d31aeaf'), 'datarelease') #temperature predictions
+
+# Extract the zipfiles
+ZipFile('datarelease/gages.zip', 'r').extractall('datarelease')
+ZipFile('datarelease/basins.zip', 'r').extractall('datarelease')
+ZipFile('datarelease/predictions.zip', 'r').extractall('datarelease')
+```
+
+## Set up model inputs
+
+Now create the input files that the model code will ingest:
+```
+import os
+import pandas as pd
+import shapefile
+import numpy as np
+
+#site attributes
+coords_shp = shapefile.Reader('datarelease/gages.shp')
+coords = pd.DataFrame(coords_shp.records(), columns=['site_no', 'site_name', 'lat', 'long']).\
+    rename(columns={'long': 'lon'})
+attr = pd.read_csv('datarelease/basin_attributes.csv', dtype={'site_no': 'str'}).\
+    merge(coords, how='outer')
+attr['site_no'] = pd.to_numeric(attr['site_no']) #attr now includes coordinates
+
+#forcing data
+temperature = pd.read_csv('datarelease/temperature_observations.csv')
+forcing = pd.read_csv('datarelease/forcings.csv')
+
+forcing_temp = forcing.merge(temperature, how = 'outer')
+forcing_temp_renamed = forcing_temp.rename(columns={"discharge(cfs)" : "00060_Mean", "wtemp(C)" : "00010_Mean", "dayl(s/d)" : "dayl(s)", "prcp(mm/d)" :"prcp(mm/day)"})
+
+```
+### Explanation of different experiments
+
+At this point, you can decide which experiment to replicate. `experiments.csv` in the data release describes the various experiments run in this data release. These columns are most important:
+
+- train_filter: Apply this filter to forcing data to filter out training and test data
+- test_filter: apply this filter to get *only* the test data. Test/train time period split is handled internally in the modeling code when running, use this to extract sites for a particular test set when checking performance.
+- hidden_sizes:  The sizes of hidden layers of the neural network.  Set these accordingly in StreamTemp-Integ.py
+- batch_size: Size of training batch.  Also set in StreamTemp-Integ.py
+
+To filter the sites and forcing data based on the first experiment in experiments.csv, and write the input files:
+```
+os.makedirs('input/forcing', exist_ok = True)
+os.makedirs('input/attr', exist_ok = True)
+attr_filtered = attr.query('dag >= 60')
```

Comment:
The first experiment in experiments.csv is 
![image](https://user-images.githubusercontent.com/12039957/129743578-1fb2da60-e990-4622-9aee-1da152bdaa1c.png)
which would have `attr.query('dag >= 99')`, wouldn't it?

Reply:
I use Jupyter Lab and an R console within Jupyter Lab, which allows me to copy-paste from the README (or other files) into the console interactively. It's not quite RStudio but works well enough for light development.